azure-pipelines.yml

```diff
@@ -162,7 +162,10 @@ stages:
       - powershell: |
           . .\scripts\utils.ps1
 
-          Remove-Item -Path analyzers\tests\SonarAnalyzer.UnitTest\TestResults\ -Recurse -Verbose
+          $TestResultsDir = "analyzers\tests\SonarAnalyzer.UnitTest\TestResults\"
+          if (Test-Path $TestResultsDir) { Remove-Item -Path $TestResultsDir -Recurse -Verbose }
```

Comment:
this is necessary when the system is dirty (rerun?)

![image](https://user-images.githubusercontent.com/38876598/180215245-63a0fbdd-5c95-4d3e-91af-f04ecb02dab0.png)

Reply:
actually now that I use a specific name for the TRX it's not a problem anymore